src/components/Elements.test.js

```diff
@@ -76,12 +78,12 @@ describe('Elements', () => {
     );
     const childContext = wrapper.node.getChildContext();
 
-    const mockCallback = jest.fn();
-    childContext.addElementsLoadListener(mockCallback);
-    expect(mockCallback).toHaveBeenCalledTimes(1);
-    expect(mockCallback).toHaveBeenLastCalledWith(true);
-    childContext.addElementsLoadListener(mockCallback);
-    expect(mockCallback).toHaveBeenCalledTimes(2);
-    expect(mockCallback).toHaveBeenCalledWith(true);
+    childContext.addElementsLoadListener(first => {
+      expect(first).toEqual(true);
+    });
+    childContext.addElementsLoadListener(second => {
+      expect(second).toEqual(true);
+      done();
+    });
```

Comment:
I'm a little concerned whether we are sure the callbacks will always be run, and in this order. Otherwise we might call `done` before the first assertion has even run.